tools/build_chain.sh

```diff
@@ -927,6 +927,16 @@ node=\$(basename \${NODE_FOLDER})
 node_pid=${ps_cmd}
 if [ -n "\${node_pid}" ];then
     echo "\${node} is trying to load new groups. Check log for more information."
+    DATA_FOLDER=${NODE_FOLDER}/data
+    for dir in $(ls ${DATA_FOLDER})
+    do
+        if [[ -d "${DATA_FOLDER}/${dir}" ]] && [[ -n "$(echo ${dir} | grep -E "^group\d+$")" ]]; then
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5420870)